bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/protobuf/BigtableZeroCopyByteStringUtil.java

```diff
@@ -12,6 +12,29 @@
  * @version $Id: $Id
  */
 public final class BigtableZeroCopyByteStringUtil {
+
```

Comment:
UnsafeByteOperations.unsafeWrap(ByteBuffer.wrap(byte[])) would end up using BoundedByteString which isn't ideal.  LiteralByteString is better when you're using the whole byte[] rather than a segment of it.